homeassistant/components/home_connect/sensor.py

```diff
@@ -69,9 +76,15 @@ async def async_update(self) -> None:
                 # if the date is supposed to be in the future but we're
                 # already past it, set state to None.
                 self._attr_native_value = None
-            else:
+            elif status[BSH_OPERATION_STATE][ATTR_VALUE] in [
```

Comment:
Do we know that the `ATTR_VALUE` key is in the dict? Below on line 89 we use `dict.get` to look it up. Does it mean that that key is optional and we need to consider that too here?

Reply:
From my tests I'm quite sure that the BSH_OPERATION_STATE and ATTR_VALUE  for it are always in the dict because the appliance has always an operation state. But I can add a check.